lib/pbench/test/unit/server/query_apis/test_controllers_list.py

```diff
@@ -129,6 +131,7 @@ def test_query(
         find_template,
         build_auth_header,
         user,
+        pbench_token,
```

Comment:
`pbench_token` is a requirement for `build_auth_header`, although I think it includes that itself. Or maybe it didn't, and I added it later. :-(

Reply:
Not to state the obvious or anything, but it seems like if the `build_auth_header` fixture requires `pbench_token`, it should reference it itself rather than depending on the referencing test to reference it.  (God, how I love pronouns!)